src/features/content-explorer/item-list/ItemListButton.js

```diff
@@ -38,6 +45,7 @@ const ItemListButton = ({ contentExplorerMode, id = '', isDisabled = false, isSe
 ItemListButton.propTypes = {
     contentExplorerMode: ContentExplorerModePropType.isRequired,
     id: PropTypes.string,
+    isChecked: PropTypes.bool,
```

Comment:
I am using isChecked in the itemButtonRenderer function in the parent component in admin console because item buttons cannot be both selected and disabled if you are using isDisabled and isSelected

Reply:
There is a test demonstrating this at line 47 in the ItemListButton.test.js